README.md

```diff
@@ -4,68 +4,68 @@ jrnl [![Build Status](https://travis-ci.com/jrnl-org/jrnl.svg?branch=master)](ht
 _To get help, [submit an issue](https://github.com/jrnl-org/jrnl/issues/new) on
 Github._
 
-*jrnl* is a simple journal application for your command line. Journals are
-stored as human readable plain text files - you can put them into a Dropbox
-folder for instant syncing and you can be assured that your journal will still
-be readable in 2050, when all your fancy iPad journal applications will long be
-forgotten.
+`jrnl` is a simple journal application for the command line.
 
-Optionally, your journal can be encrypted using the [256-bit
-AES](http://en.wikipedia.org/wiki/Advanced_Encryption_Standard).
+Its goal is to facilitate the rapid creation and viewing of journal entries. It
+is flexible enough to support different use cases and organization strategies.
+It is powerful enough to search through thousands of entries and display, or
+"filter," only the entries you want to see.
 
-### Why keep a journal?
+`jrnl` includes support for [128-bit AES
+encryption](http://en.wikipedia.org/wiki/Advanced_Encryption_Standard) using
+[cryptography.Fernet](https://cryptography.io/en/latest/fernet/).
```

Comment:
The 128-bit correction is great, but could you please remove the cryptography.Fernet reference? It's a bit much for the first section of a readme file.